src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumbCustomRendererValue.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {Meta} from 'app/types';
+import AnnotatedText from 'app/components/events/meta/annotatedText';
+
+type Props = {
+  meta?: Meta;
+  value: React.ReactElement | string;
+};
+
+const BreadcrumbCustomRendererValue = ({value, meta}: Props) => {
+  if (!meta) {
+    return typeof value === 'string' ? <React.Fragment>{value}</React.Fragment> : value;
```

Comment:
Why do we need to wrap `value` with a `Fragment` if it's a string?

Reply:
because a React Function component expects the return to be null or a React Element

Reply:
I was getting type errors, but I believe it was just because my ts compiler is a bit slow...now it's fine :)

Reply:
![image](https://user-images.githubusercontent.com/29228205/74815119-ef659380-52f8-11ea-8ccb-a474e843df78.png)

React components expect the return to be `null` or `React Element`

Reply:
I have converted the React Component to function :-)